dz-14-xpath/src/test/java/Buttons.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.JavascriptExecutor;
+
+public class Buttons {
+    public static void main(String[] args) {
+        System.setProperty("web-driver.chrome.driver", "chromedriver.exe");
```

Comment:
You set webdriver incorrectly, property name must be `webdriver.chrome.driver`. But you do not need to do that, use `WebDriverManager`